apps/frontend/src/libs/hooks/use-search-reset/use-search-reset.hook.ts

```diff
@@ -0,0 +1,13 @@
+import { useAppForm, useCallback } from "~/libs/hooks/hooks.js";
+
+const useSearchReset = (defaultSearch: string): (() => void) => {
+	const { handleReset } = useAppForm({
```

Comment:
We don't need to use separate useAppForm, we need to use useAppForm that controls the Search. I see it's defined in the search itself, we need to move it out of Search component and pass control and errors to it.

Reply:
Gotcha, will fix it right away

Reply:
I made it work, but gotta refactor a little bit for better reusability